icons/folder-acf.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" clip-rule="evenodd" fill-rule="evenodd" stroke-linejoin="round" stroke-miterlimit="1.41" viewBox="0 0 24 24"><path d="M10 4H4c-1.11 0-2 .89-2 2v12c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V8a2 2 0 0 0-2-2h-8z" fill="#00e4bc" fill-rule="nonzero"/><path d="m16.911 8.83-1.033.035-4.717 11.966 3.168-.018s.43-1.136.654-1.687c1.309 0 2.617.017 3.926.017.223.551.654 1.688.654 1.688l3.357-.052-4.976-11.914zm.018 5.028.895 2.428c-.585.017-1.17.017-1.739.017z" fill="#ccfff6" style="stroke-width:1.72166"/></svg>
```

Comment:
Thank you, the icon looks great! There is only one last thing that would be important. Could please check that you're using colors from the Material Color Palette? You can simply paste your colors into the [Material Color Converter](https://pkief.github.io/material-color-converter/) and select a similar color from the palette.